app/layout.tsx

```diff
@@ -15,7 +17,10 @@ export default function RootLayout({
 }) {
   return (
     <html lang="en">
-      <body className={inter.className}>{children}</body>
+      <body className={inter.className}>
+        <ThemeSwitcher />
```

Comment:
TODO: Find a way to integrate it that is not obnoxious. Maybe a keystroke that toggles it or hidden in a menu?